File: src/awt2swing/Panel.java
package awt2swing;

import java.awt.LayoutManager;

import javax.swing.JPanel;

public class Panel extends JPanel {

	public Panel(LayoutManager layout) {
		super(layout);
	}

	public void setName(String name) {
		System.out.println(name);
		super.setName(name);

	}
	public Panel() {
		super();
	}
	
//	public void addMouseListener(MouseListener c) {
//		//super.addMouseListener(c);
//		
//	}
//	public void addMouseMotionListener(MouseMotionListener c) {
//		//super.addMouseMotionListener(c);
//	}
}

File: site/swingjs/j2s/swingjs/awt/Panel.js
Clazz.declarePackage ("swingjs.awt");
Clazz.load (["javax.swing.JPanel"], "swingjs.awt.Panel", null, function () {
c$ = Clazz.declareType (swingjs.awt, "Panel", javax.swing.JPanel);
Clazz.defineMethod (c$, "setName", 
function (name) {
System.out.println ("Panel " + name);
Clazz.superCall (this, swingjs.awt.Panel, "setName", [name]);
}, "~S");
});
